### PR TITLE
adds vadr 1.4.2

### PR DESCRIPTION
**Pull request by kapsakcj**
Adds dockerfile for VADR 1.4.2. Dockerfile now includes a test layer. Builds successfully

~~Still need to add README.md~~

Pull Request (PR) checklist:
- [x] Include a description of what is in this pull request in this message.
- [x] The dockerfile successfully builds to a test target for the user creating the PR. (i.e. `docker build --tag samtools:1.15test --target test docker-builds/samtools/1.15` )
- [x] Directory structure as name of the tool in lower case with special characters removed with a subdirectory of the version number (i.e. `spades/3.12.0/Dockerfile`)
   - [x] (optional) All test files are located in same directory as the Dockerfile (i.e. `shigatyper/2.0.1/test.sh`)
- [x] Create a simple container-specific [README.md](https://github.com/StaPH-B/docker-builds/blob/master/.github/workflow-templates/readme-template.md) in the same directory as the Dockerfile (i.e. `spades/3.12.0/README.md`)
   - [x] If this README is longer than 30 lines, there is an explanation as to why more detail was needed
- [x] Dockerfile includes the recommended [LABELS](https://github.com/StaPH-B/docker-builds/blob/master/dockerfile-template/Dockerfile#L8-L18) 
- [x] Main [README.md](https://github.com/StaPH-B/docker-builds/blob/master/README.md) has been updated to include the tool and/or version of the dockerfile(s) in this PR
- [x] [Program_Licenses.md](https://github.com/StaPH-B/docker-builds/blob/master/Program_Licenses.md) contains the tool(s) used in this PR and has been updated for any missing

### CODE REVIEW

File: .github/workflow-templates/readme-template.md [hunk 1]
```diff
@@ -9,8 +9,8 @@ Full documentation: link to documentation or wiki
 
 <brief description>
 
-# Example Usage
+## Example Usage
 
-```
+```bash
 <how creator uses it>
```
Comment (erinyoung) [inline .github/workflow-templates/readme-template.md:15]: Why the change in the readme template?

File: vadr/1.4.2/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+ rm -rf /vadr-models-sarscov2* && \
+ cp -nv ${VADRINSTALLDIR}/vadr-models-calici/* ${VADRMODELDIR} && \
+ rm -rf ${VADRINSTALLDIR}/vadr-models-calici/
+
+# Virus model files other than sarscov2 will need to be made available to vadr either in
+# the $VADRMODELDIR or another path can be specified using the 'v-annotate.pl -mdir' option.
+# These files will need to be mounted into the container at runtime, e.g. 'docker run -v' option.
+
+# set working directory
+WORKDIR /data
```
Comment (erinyoung) [inline vadr/1.4.2/Dockerfile:78]: Looks like the same from 1.4.1

File: vadr/1.4.2/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+ /opt/vadr/vadr/miniscripts/fasta-trim-terminal-ambigs.pl \
+        /test-data/SRR13957123.consensus.fa \
+        --minlen 50 \
+        --maxlen 30000 \
+        > /test-data/SRR13957123.consensus.trimmed.fasta && \
+ v-annotate.pl --noseqnamemax --glsearch -s -r --nomisc \
+       --mkey sarscov2 --lowsim5seq 6 --lowsim3seq 6 --alt_fail lowscore,insertnn,deletinn \
+       "/test-data/SRR13957123.consensus.trimmed.fasta" \
+       "SRR13957123-vadr-outdir" && \
+  ls SRR13957123-vadr-outdir
```
Comment (erinyoung) [inline vadr/1.4.2/Dockerfile:100]: This test 
1. downloads the consensus fasta from this github repo
2. runs the built-in vadr tests
3. runs the vadr workflow on the downloaded fasta

Then shows off the final directory

File: .github/workflow-templates/readme-template.md (hunk 1 above)
Comment (kapsakcj) [reply .github/workflow-templates/readme-template.md:15]: VSCode Markdown linting always shows me an formatting warning when there are two level 1 headings (lines that begin with one `#`) in the same markdown document. It looks a bit cleaner when there is only one level 1 heading at the top, and all those below are level 2 or lower headings

the `bash` bit is just so that the rendered code has bash syntax highlighting

Just a style thing.

**Review by erinyoung — APPROVED** [review]
It looks good to me.

File: .github/workflow-templates/readme-template.md (hunk 1 above)
Comment (erinyoung) [reply .github/workflow-templates/readme-template.md:15]: Sounds good to me

**Comment by erinyoung** [comment]
No changes were requested. I'm going to merge and deploy to both docker and quay and replace the "latest" tag.
